harpy/bin/molecule_coverage.py

```
@@ -116,32 +90,21 @@ def print_windowed_depth(contig, window):
-                print_windowed_depth(LASTCONTIG, args.window)
-            # create/reset database
-            conn = sqlite3.connect(':memory:')
-            cursor = conn.cursor()
-            cursor.execute('''
-                CREATE TABLE number_counts (
-                    number INTEGER PRIMARY KEY,
-                    count INTEGER
-                )
-            ''')
-            initialize_contig(contigs[contig])
+                print_depth_counts(LASTCONTIG, counter, geno_intervals)
```

Ruff (0.8.2)

Undefined name `counter`

(F821)